core/controllers/acl_decorators.py

```diff
@@ -117,13 +117,13 @@ def test_is_source_mailchimp(self, secret, **kwargs):
         """
         if feconf.MAILCHIMP_WEBHOOK_SECRET is None:
             raise self.PageNotFoundException
-        elif secret != feconf.MAILCHIMP_WEBHOOK_SECRET:
+        if secret != feconf.MAILCHIMP_WEBHOOK_SECRET:
```

Comment:
Suggest adding a newline above this one for readability, since they're now two separate "things" rather than a single if/elif/else. 

Ditto elsewhere, where relevant.

Reply:
Done!